Reactor/Networking/HandshakePopup.cs

```diff
@@ -0,0 +1,16 @@
+namespace Reactor.Networking;
+
+internal static class HandshakePopup
+{
+    public const string Message =
+        """
+        This server doesn't support modded handshake.
+        The lobbies shown may be incompatible with your current mods.
```

Comment:
```suggestion
        The lobbies shown could be incompatible with your current mods.
```
I think this improves the text flow even more
![2023-12-08T19:29:46,682761595+01:00](https://github.com/NuclearPowered/Reactor/assets/5243971/7c365af1-e410-4d8a-ab98-6f19dfa3e758)